lib/systems/include/SteeringSystem.h

```diff
@@ -0,0 +1,136 @@
+#ifndef STEERINGSYSTEM
+#define STEERINGSYSTEM
+#include <math.h>
+#include <tuple>
+#include <etl/singleton.h>
+#include "SharedFirmwareTypes.h"
+
+struct SteeringParams {
+    uint32_t min_steering_1;  // Raw ADC at leftmost (min) steering angle
+    uint32_t max_steering_1;  // Raw ADC at rightmost (max) steering angle
+    uint32_t min_steering_2;  // (optional) for redundant steering sensor
+    uint32_t max_steering_2;
+
+    uint32_t min_sensor_steering_1; // Min value that the sensor can output (if ADC reads less than this, sensor is likely unplugged)
```

Comment:
Did you double-check this behavior of the steering sensor when you unplugged it?